packages/icon-button/src/mwc-icon-button-base.ts

```diff
@@ -25,15 +28,95 @@ export class IconButtonBase extends LitElement {
 
   @property({type: String}) label = '';
 
+  @query('button') buttonElement!: HTMLElement;
+
+  @queryAsync('mwc-ripple') ripple!: Promise<Ripple|null>;
+
+  @internalProperty() protected shouldRenderRipple = false;
+
+  protected rippleHandlers: RippleHandlers = new RippleHandlers(() => {
+    this.shouldRenderRipple = true;
+    return this.ripple;
+  });
+
+  protected renderRipple() {
```

Comment:
Please do NOT update this method or CL. I have done this for you internally and updating this CL will reset the merge progress